src/components/FilterBar.jsx

```diff
@@ -1,112 +1,72 @@
-import PropTypes from 'prop-types';
 import {
   Grid,
-  FormControl,
-  Select,
+  Menu,
   MenuItem,
-  Button,
-  Popover,
+  IconButton,
+  Typography,
+  Box,
 } from '@mui/material';
-import AmountFilter from './AmountFilter';
-import GradeLevelFilter from './GradeLevelFilter';
 import useQueryParams from '../lib/useQueryParams';
 import sortOptions, { DEADLINE_ASC } from '../lib/sortOptions';
-import MajorFilter from './MajorFilter';
-import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+import TuneIcon from '@mui/icons-material/Tune';
+import PropTypes from 'prop-types';
+import ImportExportIcon from '@mui/icons-material/ImportExport';
 
-const FilterButton = ({ title, children, popoverWidth }) => {
+export default function FilterBar({ openFilter }) {
+  const [{ sortBy }, setQueryParam] = useQueryParams();
   const [anchorEl, setAnchorEl] = useState(null);
 
   return (
-    <>
-      <Button
-        variant="outlined"
-        onClick={(e) => setAnchorEl(e.currentTarget)}
-        endIcon={<ArrowDropDownIcon color="primary" />}
-        sx={{ m: 1, height: (theme) => theme.spacing(4) }}>
-        {title}
-      </Button>
-      <Popover
-        open={Boolean(anchorEl)}
-        anchorEl={anchorEl}
-        onClose={() => setAnchorEl(null)}
-        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
-        transformOrigin={{ vertical: 'top', horizontal: 'left' }}
-        PaperProps={{ style: popoverWidth }}>
-        {children}
-      </Popover>
-    </>
-  );
-};
-
-FilterButton.propTypes = {
-  title: PropTypes.string.isRequired,
-  children: PropTypes.node.isRequired,
-  popoverWidth: PropTypes.object,
-};
-
-FilterButton.defaultProps = {
-  popoverWidth: undefined,
-};
-
-export default function FilterBar() {
-  const [{ minAmount, maxAmount, grades, majors, sortBy }, setQueryParam] =
-    useQueryParams();
-
-  return (
-    <Grid
-      container
-      spacing={2}
-      justifyContent="space-between"
-      sx={{ flexGrow: 1 }}>
-      <Grid item>
-        <FilterButton title="Majors" popoverWidth={{ minWidth: 280 }}>
-          <MajorFilter
-            majors={majors}
-            onSelect={(m) => setQueryParam('majors', m)}
-            onDelete={(m) =>
-              setQueryParam(
-                'majors',
-                majors.filter((major) => major !== m)
-              )
-            }
-          />
-        </FilterButton>
+    <Box sx={{ pb: 3 }}>
+      <Grid
+        container
+        justifyContent="space-between"
+        alignItems="center"
+        direction="row-reverse"
+        sx={{ flexGrow: 1, bgcolor: 'primary.main', p: 2 }}>
+        <Grid item sx={{ display: 'flex', alignItems: 'center' }}>
+          <IconButton
+            onClick={(e) => setAnchorEl(e.currentTarget)}
+            color="secondary">
+            <ImportExportIcon />
+          </IconButton>
+          <Typography color="secondary">Sort</Typography>
+        </Grid>
```

Comment:
How about using a [button with an icon](https://mui.com/components/buttons/#buttons-with-icons-and-label) instead of this?